jena-fuseki2/jena-fuseki-ui/vite.config.js

```diff
@@ -84,7 +62,7 @@ export default defineConfig({
         target: `http://localhost:${process.env.FUSEKI_PORT || 3030}`,
         changeOrigin: true,
         secure: false,
-        ws: true,
+        ws: false,
```

Comment:
Can't recall why I enabled it, so left it as a separate commit so I remember why/where that was disabled.

In case we need that (Vite has a built-in endpoint to refresh the page in dev mode) we should be able to add it just for the endpoint `/`.